### ISSUE TITLE
Add located_in relation to subset of Molecular Entities to accommodate CC only annotations

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/go-shapes#20]
For the CC only version of the form, as well as the MOD imports, we will need to specify that a Gene, Complex, or Protein can be 'located in' a Cellular Component:

`<Gene> @<MolecularEntity>` AND {
   bl:category [GoGene:]  ;
   located_in: `@<CellularComponent>` {0,1};
}// rdfs:comment  "a gene (a piece of DNA with a purpose)"

`<Complex> @<MolecularEntity>` AND {
   bl:category [GoComplex:]  ;
    located_in: `@<CellularComponent>` {0,1};
}// rdfs:comment  "a protein complex"

`<Protein> @<MolecularEntity>` AND {
   bl:category [GoProtein:]  ;
    located_in: `@<CellularComponent>` {0,1};
}// rdfs:comment  "a protein"

**Comment by goodb** [comment]
@vanaukenk just to confirm as I was told explicitly not to do this in the reactome imports..  

This says that you could have an instance of a gene located_in a CC in a go-cam.   

I understood that we should only be using mf - occurs in - cc to capture localization information.  In cases with no mf annotation, we would just leave it as root and the path from gene to cc would still be present and be accessible in the gpad export. 

please confirm.

**Comment by vanaukenk** [comment]
@goodb

For the MOD (i.e. MGI and WB) imports, we had decided to import the conventional annotations as GP 'part of' CC (or colocalizes_with).

http://wiki.geneontology.org/index.php/Noctua_MOD_Imports#Cellular_Component

After the USC meeting in June, though, I thought we had decided to use 'located in' for the default GP to CC relation.

This relation is what is illustrated in the upper right of the first figure in the Google doc:

https://docs.google.com/document/d/1OsE19zh8KE_2wT3-8oJysqEGqy-fahnhcjthO34IWP4/edit

I think for now we want to keep this, since with our conventional annotations curators didn't limit CC annotations solely to those where the GP was known, or presumed, to enable its function.

Happy to discuss this further on another call, though.

**Comment by pgaudet** [comment]
Yes we do want to keep the 'located_in'. And for the Reactome imports, since every entity has a function, AFAIK, it made sense to use 'occurs_in'. 

Pascale

**Comment by deustp01** [comment]
Full disclosure - every Reactome entity SHOULD have a function. Two limitations - not all those functions are captured as annotations with GO MF terms - binding, for example, is not. Second, the requirement for entities to have functions is not enforced 100% of the time, so there are probably a few cases where we not only did not assign a function, but there really isn't one. There should be a workaround for the binding events because they can be recognized as such even in the absence of a binding molecular function term; the others should be rare and QA on Reactome-derived GO-CAM models will find them.

**Comment by pgaudet** [comment]
@deustp01 In those cases, are the proteins annotated to GO Processes ? 
If that's the case, this would be acceptable in the current version of the specs, with MF = root. 

Pascale

**Comment by deustp01** [comment]
A process annotation is not guaranteed. Process annotations are not required in Reactome, but they are always appropriate, so an approach here might be for the relevant script to complain when it can't find either a function or a process attribute (complaints should be infrequent) and pass the complaints on to me.

**Comment by vanaukenk** [comment]
I'm re-opening this ticket, because we have some inconsistency in the ShEx wrt entities and their relations to CC.

<InformationBiomacromolecule> @<ChemicalEntity> AND EXTRA a {
   a @<InformationBiomacromoleculeClass> +;
   part_of: @<CellularComponent> {0,1};
   part_of: @<ProteinContainingComplex> {0,1};
}// rdfs:comment  "an information biomacromolecule - e.g. a protein or RNA product"

<Protein> @<ChemicalEntity> AND EXTRA a {
  a @<ProteinClass> ;
  located_in: @<CellularComponent> {0,1};
}// rdfs:comment  "a protein"

<ProteinContainingComplex> EXTRA a {
  a @<ProteinContainingComplexClass>;
  located_in: @<CellularComponent> {0,1};
  has_part: @<InformationBiomacromolecule> *;
} // rdfs:comment  "a protein complex"

Note that we removed the 'Gene' shape earlier this week, as it was redundant with the information we have in the 'InformationBiomacromolecule' shape.

https://github.com/geneontology/go-shapes/pull/175

I propose two things:

1.  We remove the 'Protein' shape as it is also redundant with 'InformationBiomacromolecule'.
2.  For the 'ProteinContainingComplex' shape, we change the 'located in' <CellularComponent> to 'part of'.  This is consistent with what is in the ontology and the Noctua Form.

@ukemi @goodb

**Comment by goodb** [comment]
@vanaukenk this seems fine to me.  If we ever find a case where we treat proteins different from genes we can add the protein shape back.  

(Sure wish you folks in the consortium could have agreed on one (gene/protein) or the other .. can't imagine how many hours that would have saved over the years - such is life!)
